packages/astro/src/content/vite-plugin-content-assets.ts

```diff
@@ -34,7 +34,7 @@ export function astroDelayedAssetPlugin({ mode }: { mode: string }): Plugin {
 			if (isDelayedAsset(id)) {
 				const basePath = id.split('?')[0];
 				const code = `
-					export { Content, getHeadings, _internal } from ${JSON.stringify(basePath)};
+					export { Content, getHeadings } from ${JSON.stringify(basePath)};
```

Comment:
It seems `frontmatter` needed to be added here where `_internal` was removed.

- #5927